packages/framework/src/Component/UploadedFile/UploadedFileFacade.php

```diff
@@ -98,25 +101,26 @@ public function manageFiles(object $entity, UploadedFileData $uploadedFileData,
      * @param object $entity
      * @param string $entityName
      * @param string $type
-     * @param array $temporaryFilenames
+     * @param string $temporaryFilename
+     * @param string $uploadedFileName
      */
-    protected function uploadFile(object $entity, string $entityName, string $type, array $temporaryFilenames): void
+    protected function uploadFile(object $entity, string $entityName, string $type, string $temporaryFilename, string $uploadedFileName): void
     {
-        if (count($temporaryFilenames) > 0) {
-            $entityId = $this->getEntityId($entity);
+        $entityId = $this->getEntityId($entity);
+
+        $this->deleteAllUploadedFilesByEntity($entity);
```

Comment:
Deleting files does not belong into `uploadFIle()` for me.

Reply:
This has not been changed in this PR

Reply:
But it could be fixed in this one. Do not you think?

Reply:
Fixed